docs/sources/configuration/_index.md

```diff
@@ -331,6 +331,12 @@ The query_frontend_config configures the Loki query-frontend.
 # CLI flag: -querier.max-outstanding-requests-per-tenant
 [max_outstanding_per_tenant: <int> | default = 100]
 
+# If a querier disconnects without sending notification about graceful shutdown,
+# the query-frontend will keep the querier in the tenant's shard until the forget
+# delay has passed. This feature is useful to reduce the blast radius when shuffle-sharding is enabled.
```

Comment:
In addition to my suggested change, can you please implement alternate wording for "reduce the blast radius?" I don't know what you mean with that phrase, and I don't want to use incendiary wording.  I know that the Cortex documentation (https://cortexmetrics.io/docs/guides/shuffle-sharding/) uses that phrase, but we can do better.

```suggestion
# If a querier disconnects without sending a notification about graceful shutdown,
# the query frontend will keep the querier in the tenant's shard until this forget
# delay has passed. This feature is useful to reduce the blast radius when shuffle sharding is enabled.
```

Reply:
I got all the descriptions from the code (cli flag descriptions), will rephrase that

Reply:
made it a bit lengthy but it's a complex option